dashboard/src/app/(authenticated)/(diagram)/components/Modals/UserTaskRun.tsx

```diff
@@ -82,7 +82,7 @@ export const UserTaskRun: FC<Modal> = ({ data, nodeRun, userTaskNode }) => {
                       {resultsToRender.map((result, index) => (
                         <tr key={index} className="border-b border-neutral-200">
                           <td className="px-6 py-4">{result.field}</td>
-                          <td className="px-6 py-4">{result.value}</td>
+                          <td className="px-6 py-4">{result.value?.toString()}</td>
```

Comment:
Is this necessary? cause this should automatically turn into a string?

Reply:
not bool values, True or False